src/clients/customers/search/search.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { RestClient } from '@utils/restClient';
+import { MercadoPagoConfig } from '@src/mercadoPagoConfig';
+import search from '.';
+import { CustomerSearchOptions } from './types';
```

Comment:
para manter o padrão, coloca o import do modulo testado primeiro, depois importa os modulos auxiliares (se tiver) e por ultimo, o import das tipagens (se tiver), além disso, quando for importar tipo, lembrar de colocar o prefixo `type` nas importação

Reply:
feito